website/docs/components/inputs/cork.md

```diff
@@ -0,0 +1,91 @@
+---
+title: cork
+type: input
+status: beta
+categories: ["Utility"]
+---
+
+<!--
+     THIS FILE IS AUTOGENERATED!
+
+     To make changes please edit the contents of:
+     lib/input/cork.go
+-->
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::caution BETA
+This component is mostly stable but breaking changes could still be made outside of major version releases if a fundamental problem with the component is found.
+:::
+Use signals from one input to suspend or resume reads from another input
+
+```yaml
+# Config fields, showing default values
+input:
+  label: ""
+  cork:
+    initially_corked: false
+    input: null
+    signal: null
```

Comment:
I'm guessing there's a bug in how the default (or zero value of NewInputField?) is rendered to markdown?

Reply:
Yeah this is a docs thing, harmless for now but need to decide what to put in as a placeholder for docs examples.